app/src/main/java/com/arqoders/hotelapp/data/server/TheEntityDbResult.kt

```diff
@@ -0,0 +1,25 @@
+package com.arqoders.hotelapp.data.server
```

Comment:
Recomendaría que estas clases estén en sus respectivos archivos y no tener un archivo que agrupe muchas clases

Reply:
I split up this file into SearchResponse class, Group class and Entity class